incubator/zookeeper/templates/ss.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: "{{.Release.Name  | trunc 24 }}"
```

Comment:
Do you mind if I add the suffix zk to this? Otherwise its hard to figure out which pods belong to ZK within the release.